Src/java/qdm/schema/qdm.5.3.xsd

```diff
@@ -272,6 +272,7 @@
 					<xs:element name="method" type="dt:Code" minOccurs="0" maxOccurs="1"/>
 					<xs:element name="result" type="dt:Any" minOccurs="0" maxOccurs="1"/>
 					<xs:element name="component" type="Component" minOccurs="0" maxOccurs="unbounded"/>
+					<xs:element name="relatedTo" type="dt:Code" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
There should probably be documentation on how QDM is realized in CQL, yes. We don't have anywhere that talks about this specifically right now, but the Formatting and Usage Guide is probably the appropriate place for that. I'll bring it up with the QDM stewards and see what we can do about that type of documentation.